cmake/modules/FindTensorFlow.cmake

```diff
@@ -118,12 +130,12 @@ else()
     set(_TensorFlow_TEST_VERSIONS
         "${TF_FIND_VERSION_MAJOR}.${TF_FIND_VERSION_MINOR}.${TF_FIND_VERSION_PATCH}"
         "${TF_FIND_VERSION_MAJOR}.${TF_FIND_VERSION_MINOR}")
-  else(TF_FIND_VERSION_EXACT)
+  else()
```

Comment:
I would like to keep  `else(TF_FIND_VERSION_EXACT)` to visually indicate where the branch belongs to.

Reply:
> I would like to keep `else(TF_FIND_VERSION_EXACT)` to visually indicate where the branch belongs to.

Pat.  Your suggestion is against the CMake best practices guidelines.  The CMake developers have purged that formatting from their codebases many many years ago.  This formatting option is only preserved for backward compatibility.   In fact, I am using a script written by Brad King (original and primary CMake developer) to remove those sections.

Can I propose an alternative solution that provides both visual clarities, and conforms to the modern CMake style recommendations.

```
else() # TF_FIND_VERSION_EXACT
```

Reply:
NOTE:  Anything in the endif() or else() parenthesis is ignored!

This will generate configure time warnings if not using "-Wno-dev", but is otherwise valid:
```
if(DO_SOMETHING)
  message(STATUS "A")
else( GOBBLE )
  message(STATUS "B")
endif( WOBLE )
```